test/src/main/java/org/apache/accumulo/test/fate/FateStoreIT.java

```diff
@@ -683,8 +689,12 @@ protected void testListFateKeys(FateStore<TestEnv> store, ServerContext sctx) th
     var fateKey3 = FateKey.forCompactionCommit(cid1);
     var fateKey4 = FateKey.forCompactionCommit(cid2);
 
+    // use one overlapping extent and one different
+    var fateKey5 = FateKey.forMerge(extent1);
```

Comment:
Not a change for this PR.  Was looking around to see if FateKey had any unit test that should be updated and did not see any.  Could be useful to add some unit test to check equals, hashcode, serder, etc in another PR.